packages/core/integration-tests/test/integration/require-async/ts-interop-arrow.js

```diff
@@ -0,0 +1,9 @@
+"use strict";
```

Comment:
Maybe a comment where this is from? Though the test description probably gets that across.

Might be a useful reference if this ever changes, though.